enigmatry-entry-blueprint-app/src/app/core/auth/auth.interceptor.spec.ts

```diff
@@ -1,6 +1,6 @@
-import { HTTP_INTERCEPTORS, HttpErrorResponse, HttpStatusCode } from '@angular/common/http';
-import { HttpClientTestingModule, TestRequest } from '@angular/common/http/testing';
```

Comment:
HttpClientTestingModule is deprecated was replaced with:

 provideHttpClient(withInterceptorsFromDi()),
      provideHttpClientTesting(),